Main.java

```diff
@@ -9,7 +11,23 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // create a Scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    // ask user for two integers
+    System.out.println("Please enter an integer:");
+    int firstInt = input.nextInt();
+
+    System.out.println("Please enter another integer:");
+    int secondInt = input.nextInt();
+
+    if (firstInt > secondInt) {
+      System.out.println("Your numbers in ascending order are " + secondInt + ", " + firstInt);
+    } else if (firstInt == secondInt) {
+      System.out.println("Your numbers in ascending order are " + firstInt + ", " + secondInt);
+    } else {
+      System.out.println("Your numbers in ascending order are " + firstInt + ", " + secondInt);
```

Comment:
Make sure that you are adding comments to your code. Think of the steps your program makes and turn these into comments. Otherwise, excellent work!